src/scss/cs/competition.scss

```diff
@@ -0,0 +1,51 @@
+.ioe {
+  &-row {
+    gap: 40px;
+    justify-content: center;
+  }
+
+  &-col {
+    background-color: white;
+  }
+
+  &-custom-width {
+    width: 90%;
+  }
+
+  @media (min-width: 1200px) {
+    &-custom-width {
+      width: 540px;
+    }
+  }
+
+  &-title {
+    color: #120540;
+    font-size: 24px;
+  }
+
+  &-text {
+    color: #1c1c1c;
+    font-size: 19px;
+  }
+
+  &-comment {
```

Comment:
the vast majority of this is bootstrappable.

Reply:
Changed to using bootstrap